openquake/hazardlib/gsim/bindi_2014.py

```diff
@@ -114,6 +114,34 @@ def _get_style_of_faulting_term(C, ctx):
     return C["sofN"] * NS + C["sofR"] * RS + C["sofS"] * SS
 
 
+def _get_dist_type(GMPE):
+    """
+    Get distance type required for the corresponding class of 
+    Bindi et al. (2014)
+    """
+    # If gsim is [BindiEtAl2014RjbArmenia] strip to [BindiEtAl2014Rjb]
+    toml_str = GMPE._toml.split('\n')[0]
```

Comment:
This is horrible. Why do you need to access a private attribute? Why not simply using GMPE.__class__.__name__? Also all caps are reserved to constants, you should use `gmpe` as variable name.